cmd/iam.go

```diff
@@ -532,6 +532,12 @@ func (sys *IAMSys) DeletePolicy(ctx context.Context, policyName string, notifyPe
 		return errServerNotInitialized
 	}
 
+	for _, v := range iampolicy.DefaultPolicies {
+		if v.Name == policyName {
+			return fmt.Errorf("inbuilt policy `%s` not allowed to be deleted", policyName)
+		}
+	}
+
```

Comment:
this may be a bit more than we can chew, since we actually allowed default policies to be modified? - we should check if the default policies were persisted or not - if they are persisted we must allow them to be deleted().

Reply:
ok